src/pydicom/pixels/decoders/native.py

```diff
@@ -122,7 +127,12 @@ def _rle_decode_frame(
         green then all blue, with the bytes for each pixel ordered from
         MSB to LSB when reading left to right).
     """
-    if nr_bits % 8:
+    if nr_bits == 1 and nr_samples != 1:
+        raise NotImplementedError(
+            "Unable to decode RLE encoded pixel data with Bits Allocated = 1 "
```

Comment:
`(0028, 0100) 'Bits Allocated'` please

Reply:
Addressed in [e9ea3de](https://github.com/pydicom/pydicom/pull/2216/commits/e9ea3deb538f6786f5a80e0e826d7aaf7a40678c)